components/splitter/hooks/useResize.ts

```diff
@@ -25,7 +25,8 @@ export default function useResize(
 
   // Real px sizes
   const [cacheSizes, setCacheSizes] = React.useState<number[]>([]);
-
+  // cache collapsed size
+  const [collapsedSizes, setCollapsedSizes] = React.useState<number[]>([]);
```

Comment:
When `containerSize` changed, the cached data should be cleared since it's ood.

Reply:
Thank you for the reminder, But after local testing, it was found that without cleaning, the issue of incorrect proportions does not arise.